lib/http3/server.c

```diff
@@ -1500,8 +1505,71 @@ h2o_http3_conn_t *h2o_http3_server_accept(h2o_http3_server_ctx_t *ctx, quicly_ad
     return &conn->h3;
 }
 
+static void graceful_shutdown_close_stragglers(h2o_timer_t *entry)
+{
+    h2o_context_t *ctx = H2O_STRUCT_FROM_MEMBER(h2o_context_t, http3._graceful_shutdown_timeout, entry);
+    h2o_linklist_t *node, *next;
+
+    /* We've sent two GOAWAY frames, close the remaining connections */
+    for (node = ctx->http3._conns.next; node != &ctx->http3._conns; node = node->next) {
```

Comment:
I think that the last expression should be `node = next`?

Reply:
👍 Good catch, thanks! I thought I simply copied this logic from H2 but for some reason I introduced a mutation.